bindings/build.rs

```diff
@@ -1,5 +1,6 @@
 fn main() {
     windows::build! {
-        Windows::Win32::UI::WindowsAndMessaging::MessageBoxW
+        Windows::Win32::UI::WindowsAndMessaging::MessageBoxW,
+        Windows::Win32::Foundation::HWND,
```

Comment:
`windows` has a new tracker that picks up dependencies like this automagically ✨. Did you run into an issue compiling the samples? Or is this change for other reasons?

Reply:
This was from a previous change I had made, but I forgot to back this out. I'll remove it.